src/Compatibility/Microsoft.DotNet.PackageValidation/build/Microsoft.DotNet.PackageValidation.targets

```diff
@@ -77,6 +77,19 @@
       </_PackageValidationIntermediateBaselineFileContent>
     </PropertyGroup>
 
+    <PropertyGroup Condition="'$(PackageValidationBaselineVersion)' == ''">
+      <_PackageValidationIntermediateBaselineFile>$(MSBuildProjectExtensionsPath)$(MSBuildProjectFile).packagevalidation.g.targets</_PackageValidationIntermediateBaselineFile>
+      <_PackageValidationIntermediateBaselineFileContent>
+<![CDATA[<Project>
+  <Target Name="ResetEnablePackageValidation" BeforeTargets="RunPackageValidation">
```

Comment:
Why do you need a separate target and a separate targets file for this? Why can't you check if the supplied `_PackageValidationBaselineVersion` is empty before the `ValidatePackage` task is invoked?